app/src/main/java/com/infomaniak/drive/data/cache/DriveInfosController.kt

```diff
@@ -125,6 +125,18 @@ object DriveInfosController {
         }
     }
 
+    fun deleteDriveIntoDriveInfo(userId: Int) {
+        getRealmInstance().use { realm ->
+            realm.executeTransaction {
+                val drives = realm.where(Drive::class.java)
+                    .equalTo(Drive::userId.name, userId)
+                    .findAll()
+
+                drives.map { it.deleteFromRealm() }
```

Comment:
`map` will return a list containing the result of what is in the lambda.
Here we don't want that, so just use a forEach : 

```suggestion
                drives.forEach(Drive::deleteFromRealm)
```